tflint failing in slim-v4.9.3 "Plugin `aws` not found in /github/home/.tflint.d/plugins"

## Describe the bug

We're running super-linter on our codebase that is largely made up of terraform files. it has just failed with error:

```shell
2022-05-24 20:26:03 [INFO]   File:[/github/workspace/repo/path/to/file.tf]
2022-05-24 20:26:03 [ERROR]   Found errors in [tflint] linter!
2022-05-24 20:26:03 [ERROR]   Error code: 1. Command output:
------
Failed to initialize plugins; Plugin `aws` not found in /github/home/.tflint.d/plugins
```
Note that we don't use AWS, we don't use any AWS terraform providers.

I note that this is running on v4.9.3:

> Status: Downloaded newer image for ghcr.io/github/super-linter:slim-v4.9.3

I have compared to a successful run of the super-linter 6 horus ago which also used tflint to lint some terraform code, it is using an earlier version of the super-linter:

> Status: Image is up to date for ghcr.io/github/super-linter:slim-v4.9.2

## Expected behavior

This error should not occur. We haven't done anything to cause it and it does look as though its been caused by the bump to v4.9.3.

Hi @ferrarimarco , I am working with @jamiet-msm on this.
This is the output of the debug logging:
`2022-05-25 08:13:09 [DEBUG]   Found errors. Error code: 1, File type: TERRAFORM_TFLINT, Error on missing exec bit: false
2022-05-25 08:13:09 [ERROR]   Found errors in [tflint] linter!
2022-05-25 08:13:09 [ERROR]   Error code: 1. Command output:
------
Failed to initialize plugins; Plugin `aws` not found in /github/home/.tflint.d/plugins
------
2022-05-25 08:13:09 [DEBUG]   Error code: 1. Command output:
------
Failed to initialize plugins; Plugin `aws` not found in /github/home/.tflint.d/plugins`

And this is the config of the ci.yml:
`  super-lint:
    name: Lint Code Base
    runs-on: [self-hosted, ubuntu, dataplatform]
    steps:
      - name: Checkout code
        uses: actions/checkout@v2
      - name: Run Super-Linter
        uses: github/super-linter/slim@v4
        env:
          DEFAULT_BRANCH: main
          LOG_LEVEL: DEBUG
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          VALIDATE_ALL_CODEBASE: false
          VALIDATE_PYTHON_ISORT: false
          VALIDATE_DOCKERFILE: false
          VALIDATE_DOCKERFILE_HADOLINT: false
          VALIDATE_GITHUB_ACTIONS: false 
          VALIDATE_SQLFLUFF: false`

I tried adding in a .tflint.hcl file with the above config and got the following error:

```shell
`2022-05-25 08:13:09 [DEBUG]   Found errors. Error code: 1, File type: TERRAFORM_TFLINT, Error on missing exec bit: false
2022-05-25 08:13:09 [ERROR]   Found errors in [tflint] linter!
2022-05-25 08:13:09 [ERROR]   Error code: 1. Command output:
------
Failed to initialize plugins; Plugin `aws` not found in /github/home/.tflint.d/plugins
------
2022-05-25 08:13:09 [DEBUG]   Error code: 1. Command output:
------
Failed to initialize plugins; Plugin `aws` not found in /github/home/.tflint.d/plugins
------`
```
Do I need to point it towards a plugin dir?

--- a/lib/functions/detectFiles.sh
+++ b/lib/functions/detectFiles.sh
@@ -514,7 +514,7 @@ function RunAdditionalInstalls() {
     #########################
     BUILD_CMD=$(
       cd "${WORKSPACE_PATH}" || exit 0
-      "tflint --init --loglevel=${TF_LOG_LEVEL}" 2>&1
+      tflint --init --loglevel="${TF_LOG_LEVEL}"  -c "${TERRAFORM_TFLINT_LINTER_RULES}" 2>&1
     )
 
     ##############

--- a/lib/functions/detectFiles.sh
+++ b/lib/functions/detectFiles.sh
@@ -514,7 +514,7 @@ function RunAdditionalInstalls() {
     #########################
     BUILD_CMD=$(
       cd "${WORKSPACE_PATH}" || exit 0
-      tflint --init --loglevel="${TF_LOG_LEVEL}"  -c "${TERRAFORM_TFLINT_LINTER_RULES}" 2>&1
+      tflint --init --loglevel="${TF_LOG_LEVEL}" -c "${TERRAFORM_TFLINT_LINTER_RULES}" 2>&1
     )
 
     ##############

--- a/lib/functions/detectFiles.sh
+++ b/lib/functions/detectFiles.sh
@@ -504,12 +504,17 @@ function RunAdditionalInstalls() {
   if [ "${VALIDATE_TERRAFORM_TFLINT}" == "true" ] && [ "${#FILE_ARRAY_TERRAFORM_TFLINT[@]}" -ne 0 ]; then
     info "Detected TFLint Language files to lint."
     info "Trying to install the TFLint init inside:[${WORKSPACE_PATH}]"
+    # Set the log level
+    TF_LOG_LEVEL="info"
+    if [ "${ACTIONS_RUNNER_DEBUG}" = "true" ]; then
+      TF_LOG_LEVEL="debug"
+    fi
     #########################
     # Run the build command #
     #########################
     BUILD_CMD=$(
       cd "${WORKSPACE_PATH}" || exit 0
-      tflint --init 2>&1
+      "tflint --init --loglevel=${TF_LOG_LEVEL}" 2>&1
     )
 
     ##############

